src/main/java/bio/terra/cda/app/util/CDAUtils.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.cda.app.util;
+
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CDAUtils {
```

Comment:
In general, the java naming convention for names that use acronyms is to capitalize the first letter only. So this would be called `CdaUtils`. But I would pick a different name, as `parseTableName()` isn't CDA specific. Maybe this is a SQL util, or a BigQuery util? Or, if the code is moved into `QueryTranslator`, it doesn't need to be in a separate utils class.

Reply:
It may not be called out, but the code is supposed to follow google's java style guide. The note on how to create camel case names describes how to handle acronyms. https://google.github.io/styleguide/javaguide.html#s5.3-camel-case